setup.py

```diff
@@ -6,7 +6,7 @@
 setup_requires: List[str] = []
 install_requires: List[str] = [
     "ase>=3.18, <4.0.0",  # Note that we require ase==3.21.1 for pytest.
-    "pymatgen",
+    "pymatgen>=2020.1.28",
```

Comment:
pymatgen's `get_neighbor_list` is relatively new, so I added version limit here.